gzsitl/gzsitl_plugin.cc

```diff
@@ -219,6 +238,26 @@ void GZSitlPlugin::OnUpdate()
         return;
     }
 
+    case ACTIVE_ROTATING: {
+
+        // Calculate the target azimuthal angle of the target in relation to
+        // the vehicle
+        double targ_ang = (180.0 / M_PI) *
+                          atan2(target_pose_rel.pos.y, target_pose_rel.pos.x);
+
+        // Change state if vehicle is already pointing at the target
+        if (fabs(targ_ang) <= GZSITL_LOOKAT_TARG_ANG_LIMIT) {
+            simstate = ACTIVE_AIRBORNE;
+            print_debug_state("state: ACTIVE_AIRBORNE\n");
+        }
+
+        // Otherwise, continue to request the rotation
+        mavserver.queue_send_cmd_long_until_ack(
+            MAV_CMD_CONDITION_YAW, fabs(targ_ang),
+            GZSITL_LOOKAT_ROT_SPEED_DEGPS, -copysign(1, targ_ang), 1, 0, 0, 0,
+            2000);
```

Comment:
It's a "retry interval", in ms. These calls to queue_send_cmd_long_until_ack shall all be encapsulated inside the new mavserver (mav_vehicles) on my next pr, but I'll fix it anyway adding a #define for this constant.